pkg/services/object/acl/eacl/types.go

```diff
@@ -6,16 +6,16 @@ import (
 	bearer "github.com/nspcc-dev/neofs-api-go/v2/acl"
 )
 
-// Storage is the interface that wraps
-// basic methods of extended ACL table storage.
-type Storage interface {
-	// GetEACL reads the table from the storage by identifier.
+// Source is the interface that wraps
+// basic methods of extended ACL table source.
+type Source interface {
```

Comment:
Unnecessary change. Previous naming is completely fine.

Reply:
it sounds like implementation *must* store tables. so i found it strange to name it "storage" if it may be used not in cache context(does not store)

also, i wanted to rename similar interface in `netmap` but... it is already named as "Source"

roll back?